NUnit.proj

```diff
@@ -407,22 +414,12 @@
     <Message Text="* Creating the binary $(Configuration) package as $(PackageName)$(ConfigSuffix).zip" />
     <Message Text="******************************************************************" />
 
-    <CallTarget Targets="_RemoveGitDirFromImage" />
```

Comment:
Can the target `_RemoveGitDirFromImage` be removed? I don't see any other references to it.